src/Pickles/Pickles.DocumentationBuilders.Html/HtmlScenarioOutlineFormatter.cs

```diff
@@ -122,6 +123,7 @@ private XElement FormatExamples(ScenarioOutline scenarioOutline)
                         new XAttribute("class", "examples"),
                         new XElement(this.xmlns + "h3", "Examples: " + example.Name),
                         this.htmlDescriptionFormatter.Format(example.Description),
+                        (example.Tags == null || example.Tags.Count == 0) ? null : new XElement(this.xmlns + "span", HtmlScenarioFormatter.CreateTagElements(example.Tags.OrderBy(t => t).ToArray(), this.xmlns)),
```

Comment:
In order to be consistent with the formatting of tags at other places, could you please use a `<p class="tags">` - the code should be similar to https://github.com/picklesdoc/pickles/blob/develop/src/Pickles/Pickles.DocumentationBuilders.Html/HtmlScenarioFormatter.cs#L59

Better yet: refactor the various places where tags are added so they all call the same method.